ui/components/component-library/button-icon/button-icon.tsx

```diff
@@ -23,6 +23,7 @@ const buttonIconSizeToIconSize: Record<ButtonIconSize, IconSize> = {
   [ButtonIconSize.Sm]: IconSize.Sm,
   [ButtonIconSize.Md]: IconSize.Md,
   [ButtonIconSize.Lg]: IconSize.Lg,
+  [ButtonIconSize.Xs]: IconSize.Xs,
 };
```

Comment:
Adding support for Xs size for `ButtonIconSize`

Reply:
Might get some pushback on adding Xs size.

Will need to ask design first @Akatori-Design @justincheungcon